packages/frontend-shared/src/gql-components/HeaderBarContent.vue

```diff
@@ -18,56 +18,53 @@
           class="h-32px mr-18px w-32px"
           src="../assets/logos/cypress-dark.png"
```

Comment:
Completely unrelated but let's put an `alt` attr on this image while we're in this file, oops!

Reply:
Oh wow! This comment made me realize my VScode accessibility extension isn't working as expected. Thanks for the catch 🎣